FrejaEidClient/src/main/java/com/verisec/frejaeid/client/http/HttpService.java

```diff
@@ -80,6 +84,7 @@ public boolean retryRequest(IOException exception, int executionCount, HttpConte
         poolingHttpConnectionManager.setMaxTotal(20);
         poolingHttpConnectionManager.setDefaultMaxPerRoute(20);
         httpClient = httpClientBuilder.setConnectionManager(poolingHttpConnectionManager).build();
+        LOG.debug("Successfully created http client with SSL context, connection timeout {} ms and read timeout {} ms.", connectionTimeout, readTimeout);
```

Comment:
Remove space between {} for timeout and ms.

Reply:
Done.